src/flb_http_client.c

```diff
@@ -447,6 +447,19 @@ static int process_data(struct flb_http_client *c)
     return FLB_HTTP_MORE;
 }
 
+#if defined FLB_TESTS_OSSFUZZ
```

Comment:
can you make this a formal definition in CMakeLists.txt ?, like: 

```cmake
if (FLB_TESTS_OSSFUZZ)
  FLB_DEFINITION(FLB_HAVE_TESTS_OSSFUZZ)
endif()
```

so then inside the C code you do:

```c
#ifdef FLB_HAVE_TESTS_OSSFUZZ
...
#endif

that's the normal structure we use...

Reply:
Fixed!